airbyte-integrations/connectors/source-zoom/source_zoom/schemas/users.json

```diff
@@ -78,7 +87,8 @@
     },
     "verified": {
       "type": "number"
-    }
+    },
+    "display_name": { "type": "string" }
   },
   "required": []
```

Comment:
No need to specify the `required` fields for our schemas, since it could introduce the regressions.

Reply:
All `required` fields removed 👍